doc/help/embed.md

```diff
@@ -0,0 +1,271 @@
+
+Projects must have a project name.
+
+![Adding a project](adding-project.png)
+
+## Listing Projects
+
+![Listing projects](projects-listing.png)
+
+## Viewing project details
+
+![Project details](project-details.png)
+
+# Uploading Property Details
+
+You can upload a csv file to create a property with profile
+details.
+
+![Property upload](add-properties.png)
+
+To make it easier to create the csv an Excel template has been
+created. To use it, fill in the Excel spreadsheet as below and save
+the file as csv when you are done. The csv can be uploaded as in the
+picture above.
+
+## Property Details Form
+
+[This Excel spreadsheet](example-property.xlsx) has been designed to
+comprehensively capture the diverse range of building characteristics
+(house type, construction, technologies installed); plus very
+top-level info on the initial design intention and final output. The
+information in this form will be fed directly into the Embed database
+associated with each dwelling with the intention of giving external
+parties a clear view on the home, technologies installed and the
+intended/achieved performance.
+
+Information is required from the dwelling as designed, and from the
+dwelling as built, as these specifications may be different. There are
+also opportunities to add interventions that identify times when the
+performance of the dwelling may change. This could be an intervention
+in the fabric, service or a change in occupation.
+
+There are many data fields for completion, but not all will be
+relevant to your study building(s). Please attempt to complete all
+relevant fields to the best of your ability.
+
+As far as is possible fields have been pre-populated with information
+for you to select, however we fully expect there to be omissions and
+encourage the giving of additional info in boxes that request either
+'specify or give further information' boxes. These fields should only
+be used for giving clarifications - any narrative in relation to
+changes in specifications and the reasons for change, or unexpected
+challenges arising from the project must be captured in detail on the
+final reporting template document that accompanies this workbook.
+
+Some cells have comments boxes that contain further information and
+guidance. These are identified by a small red tag in the top right
+corner of the cell.
+
+### Important information:
+
+This workbook collects information from a single property in your
+study. A different datasheet must be completed for each property being
+physically monitored. If dwellings are similar then the workbook can
+be duplicated with relevant amends made to the information
+contained. Sheets do not need to be competed for those dwellings that
+are having qualitative assessments only.
+
+Most parts of the workbook should be self-explanatory. Every data
+entry cell will prompt for the information required though a pop-up
+box, which will appear when the cell is active. If there is any
```

Comment:
Are we talking now about the Excel workbook, or about the input cells in the web interface? The workbook doesn't give me pop-up prompts...